wiz-admission-controller/templates/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "wiz-admission-controller.secretApiToken" . }}
+data:
+  clientId: {{ required "Missing required value wizApiToken.clientId is required" .Values.wizApiToken.clientId | b64enc | quote }}
+  clientToken: {{ required "Missing required value: wizApiToken.clientToken is required" .Values.wizApiToken.clientToken | b64enc | quote }}
+  clientEnv: {{ required "Missing required value: wizApiToken.env is required" .Values.wizApiToken.env | b64enc | quote }}
```

Comment:
for our services we prefer not to keep tokens as exposed helm values but to write them to secret manager and use tool like External Secret Operator to import them to kubernetes as secret.

@razsha @talmarco  - WDYT about allowing to explicitly set this in the values?

Reply:
I can add "create flag" for this secret